contrib/pipelines/dummy/Makefile

```diff
@@ -0,0 +1,20 @@
+PROG := dummy
+
+SRCS := \
+	main.go \
+
+.PHONY: all
+all:
+	go build -o $(PROG) $(SRCS)
+
+.PHONY: clean
+clean:
+	rm -rf $(PROG)
+
+.PHONY: install
+install:
+	sudo cp $(PROG).yml.default /etc/skydive/$(PROG).yml
```

Comment:
Why we copy to /etc/... location ... we should use the yml from command line ... why we need this 'global' thing ?

Reply:
removed install step - I just run using the *.yml.default file directly.